Index.html

```diff
@@ -223,7 +223,7 @@ <h3>
               <li class="experince__content--right">
                 <div>
                   <div class="experince__point"></div>
-                  <div class="experince__text">
+                  <div class="experince__text experince__right--text">
```

Comment:
I'm little confused is right a specific element? or a modifier? if is a modifier it should be `experience__text--right`

Reply:
ha aha aha, I forget to eliminate that class from the element, I found a different way without using that class.